### PR TITLE
akkoma: 3.12.2 -> 3.13.1

### PR DESCRIPTION
**Pull request by tcmal**
## Description of changes

[Announcement](https://meta.akkoma.dev/t/akkoma-stable-2024-04-straight-up-fixing-it-in-the-git-and-by-it-haha-well-i-mean-our-bhackend/715) and [changelog](https://akkoma.dev/AkkomaGang/akkoma/src/branch/develop/CHANGELOG.md#2024-04).

Also updates the frontend and admin-fe packages. `admin-fe` requires a patch for `yarn.lock` to stop yarn running `git ls-remote` and failing due to no internet access: see [here](https://github.com/yarnpkg/yarn/issues/8638).

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/akkoma/admin-fe/default.nix [hunk 1]
```diff
@@ -2,26 +2,25 @@
...
 , fetchFromGitea, fetchYarnDeps
 , fixup-yarn-lock, yarn, nodejs
+, git
 , python3, pkg-config, libsass
 }:
 
 stdenv.mkDerivation rec {
   pname = "admin-fe";
-  version = "unstable-2024-02-25";
+  version = "stable-2024-04-27";
```
Comment (illdefined) [inline pkgs/servers/akkoma/admin-fe/default.nix:11]: The package naming guidelines recommend using an `unstable` prefix for packages not based on an explicit upstream release: <https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming>

**Review by illdefined — CHANGES_REQUESTED** [review]
I like that we can finally get rid of these large patches, but I suggest to take a look at the package version for `admin-fe`.

**Comment by illdefined** [comment]
Result of `nixpkgs-review pr 307324` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>akkoma</li>
    <li>akkoma-frontends.admin-fe</li>
    <li>akkoma-frontends.akkoma-fe</li>
  </ul>
</details>

**Comment by tcmal** [comment]
Okay, codespaces did something weird - sorry for the ping everyone.

**Pull request closed without merging**